src/main/java/uk/nickbdyer/tictactoe/PlayerType.java

```diff
@@ -0,0 +1,8 @@
+package uk.nickbdyer.tictactoe;
+
+public enum PlayerType {
+    H,
```

Comment:
Why not name it `Human`?
I think it would easier to understand in other contexts...